packages/common/src/search/_internal/hubSearchEventAttendees.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  GetRegistrationsParams,
+  IGetRegistrationsParams,
+} from "../../events/api";
+import {
+  getRegistrations,
+  IPagedRegistrationResponse,
+} from "../../events/api/orval/api/orval-events";
+import { eventAttendeeToSearchResult } from "../../events/api/utils/search";
+import { IPredicate, IQuery } from "../types/IHubCatalog";
+import { IHubSearchOptions } from "../types/IHubSearchOptions";
+import { IHubSearchResponse } from "../types/IHubSearchResponse";
+import { IHubSearchResult } from "../types/IHubSearchResult";
+
+export const processGetRegistrationsParams = (
+  options: IHubSearchOptions,
```

Comment:
@rweber-esri `IHubSearchOptions` doesn't have any sort of `id` property. What should we set that as?

Reply:
Ahh, I may appear to have misrepresented how the `eventId` will be passed in. I _thought_ IHubSearchOptions supported tacking other arbitrary, non-concretely typed key/values onto it for these types of cases, but that is not true. It looks like the `eventId` will be passed on the IQuery, but in it's `properties` object rather than in the `filters`. Sorry for the confusion.